mpas_analysis/ocean/nino34_index.py

```diff
@@ -154,7 +153,11 @@ def compute_nino34_index(regionSST, calendar):  # {{{
 
     anomaly = regionSST.groupby('month') - monthlyClimatology
 
-    return _running_mean(anomaly.to_pandas())  # }}}
+    # Remove the long term trend from the anomalies
+    detrendedAnomal = signal.detrend(anomaly.values)
```

Comment:
`apply` is on Dataset, not DataArray so this makes sense why the error occurs.  Let's skip for now, especially since @milenaveneziani is testing the code.